numpy/core/src/umath/ufunc_type_resolution.c

```diff
@@ -1305,8 +1305,9 @@ find_userloop(PyUFuncObject *ufunc,
             if (obj == NULL) {
                 continue;
             }
-            funcdata = (PyUFunc_Loop1d *)NpyCapsule_AsVoidPtr(obj);
-            while (funcdata != NULL) {
+            for (funcdata = (PyUFunc_Loop1d *)NpyCapsule_AsVoidPtr(obj);
```

Comment:
I'm thinking we actually want to indent these more, like we do with extended if conditions. Should probably put it in the style guide. Anyway, I'm not going to worry about that now.